cli/src/main/java/org/apache/iotdb/tool/ImportCsv.java

```diff
@@ -169,14 +172,6 @@ private static void loadDataFromCSV(File file) {
       System.out.println("CSV file read exception because: " + e.getMessage());
     } catch (IoTDBConnectionException | StatementExecutionException e) {
       System.out.println("Meet error when insert csv because " + e.getMessage());
-    } finally {
-      try {
-        if (session != null) {
-          session.close();
-        }
-      } catch (IoTDBConnectionException e) {
-        System.out.println("Sql statement can not be closed because: " + e.getMessage());
-      }
```

Comment:
Hi, if remove these lines, how to close the session?

Reply:
session will be closed in the importCsvFromFile method which calls this method. So,  closing session is unnecessary here.